src/components/DeleteModal/iindex.js

```diff
@@ -0,0 +1 @@
+export { default } from './DeleteModal';
```

Comment:
there's a small typo in the file name itself where it should be index.js instead of iindex.js